### PR TITLE
CSA-64: Implemented get all Domains endpoint

### PR DESCRIPTION
**Pull request by OleksiiKukla**

### CODE REVIEW

File: roadmap/views.py [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+
+
+class DomainListView(APIView):
+
+    def get(self, request):
+        meta_domains = MetaDomain.objects.all()
+        local_domains = LocalDomain.objects.all()
+
+        meta_domain_serializer = MetaDomainSerializer(meta_domains, many=True)
+        local_domain_serializer = LocalDomainSerializer(local_domains, many=True)
```
Comment (ssliadniev) [inline roadmap/views.py:15]: btw, do we need `flat=True` here?

File: roadmap/views.py [hunk 2]
```diff
@@ -1,3 +1,22 @@
-from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
-# Create your views here.
+from roadmap.models import LocalDomain, MetaDomain
+from roadmap.serializers import LocalDomainSerializer, MetaDomainSerializer
+
+
+class DomainListView(APIView):
```
Comment (ssliadniev) [inline roadmap/views.py]: Let's use `ListAPIView` generic (move all logic from _get_ to `list`).
Also use next pattern: {view_name}{view_class} (for instance, `DomainListAPIView`) for the view name.

File: roadmap/urls.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from roadmap.views import DomainListView
+
+app_name = "roadmaps"
+
+urlpatterns = [
+    path("domains/", DomainListView.as_view(), name="domains-get-all"),
```
Comment (ssliadniev) [inline roadmap/urls.py]: let's set next name: `domain-list-api-view`

**Review by ssliadniev — CHANGES_REQUESTED** [review]

File: roadmap/views.py [hunk 4]
```diff
@@ -1,3 +1,22 @@
...
+
+        meta_domain_serializer = MetaDomainSerializer(meta_domains, many=True)
+        local_domain_serializer = LocalDomainSerializer(local_domains, many=True)
+
+        data = {
+            "meta_domains": meta_domain_serializer.data,
+            "local_domains": local_domain_serializer.data,
+        }
+
+        return Response(data)
```
Comment (ssliadniev) [inline roadmap/views.py:22]: Do we have pagination here?

**Review by ssliadniev — CHANGES_REQUESTED** [review]

File: roadmap/views.py (hunk 2 above)
Comment (OleksiiKukla) [reply roadmap/views.py]: Good point. Done

File: roadmap/urls.py (hunk 3 above)
Comment (OleksiiKukla) [reply roadmap/urls.py]: Done

**Review by ssliadniev — APPROVED** [review]

**Pull request merged**